models/issues/milestone.go

```diff
@@ -123,6 +124,32 @@ func NewMilestone(m *Milestone) (err error) {
 		return err
 	}
 
+	sess := db.GetEngine(db.DefaultContext)
+
+	if len(m.Labels) > 0 {
+		// During the session, SQLite3 driver cannot handle retrieve objects after update something.
```

Comment:
Don't know what did you mean. I don't think we need to do that.

Reply:
This is the same as [here](https://github.com/go-gitea/gitea/blob/82ecd3b19eb1382521b7999bd0e2dd977c3b994d/models/issues/issue.go#L1022)